### PR TITLE
Vision doc

### LINKED ISSUE #85: [Feature Request]: add VISION.md
### Description

Add a VISION.md document to detail purpose and direction of this project.

### Problem

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by davidslusser**
## Pull Request

**Description:**
closes #85; adding vision doc to explain the reason for and direction of the project 

**Checklist:**
- [x] All tests pass.
- [x] Code follows the project's coding standards.
- [x] Documentation has been updated.

### CODE REVIEW

File: .github/VISION.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+### Phase Three:
+Build member profiles. With profiles, we can have some basic metrics on things like career level, geographic location, interested and expertise. This data can help provide visbility into the overall tech presence in Spokane and help drive event topics and location. This could also be a future resouce to make available to local businesses and the communiity for things like contract work, etc. (There has been some outside interest in this type of resoiurce)
+
+
+### Future goals:  
+The Spokane Tech project was started mostly by members of the Spokane Python User Group (SPUG), so natually the first version of the website is based on python. In the future the project will be re-created in other langauges/frameworks/etc. (such as Golang or Rust) as member interest dictates. This is intended to foster growth, knowledgesharing, and exposure to different tech stacks and methodologies.
```
Comment (joeriddles) [inline .github/VISION.md:36]:
```suggestion
The Spokane Tech project was started mostly by members of the Spokane Python User Group (SPUG), so naturally the first version of the website is based on Python. In the future the project may be re-created in other languages/frameworks/etc. (such as Go or Rust) as member interest dictates. This is intended to foster growth, knowledge sharing, and exposure to different tech stacks and methodologies.
```

File: .github/VISION.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+
+## Interested in participating? Great! Read on...
+
+Here are a few things you can do to get started.
+
+- Go to the site and explore. It's live at https://www.spokanetech.org
```
Comment (joeriddles) [inline .github/VISION.md:43]:
```suggestion
- Go to the site and explore. It's live at https://spokanetech.org
```

File: .github/VISION.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+- Look through the open issues and find one that interests you (issues tagged "good first issue" could be great candidates)
+https://github.com/SpokaneTech/SpokaneTech_Py/issues 
+
+- Read the docs: In our documentation directory we have design decisions, style guide, contributing guide and more. Feel free to familiarize yourself with these. 
+https://spokanetech.github.io/SpokaneTech_Py/ 
+
+- clone the repo to you machine and run locally, explore the code, break things, fix things, have fun. Step by step instructions are in the CONTRIBUTION doc.
```
Comment (joeriddles) [inline .github/VISION.md:51]:
```suggestion
- Clone the repo to you machine and run locally, explore the code, break things, fix things, have fun. Step by step instructions are in the [CONTRIBUTING](https://spokanetech.github.io/SpokaneTech_Py/CONTRIBUTING/) doc.
```

**Review by joeriddles — CHANGES_REQUESTED** [review]
This is great! Mostly of the comments are spelling changes. Moving this to the docs repo would be awesome.

File: .github/VISION.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# About the Spokane Tech Project
```
[the remark this answers is not in the document]
Comment (davidslusser) [reply .github/VISION.md:1]: sure

File: .github/VISION.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+
+## What is?
```
[the remark this answers is not in the document]
Comment (davidslusser) [reply .github/VISION.md:4]: I decided to just remove the subtitle

**Comment by davidslusser** [comment]
doc moved; spell checked working this time

**Comment by joeriddles** [comment]
Can you add the doc to `mkdocs.yml` so it appears in the docs site?

**Comment by davidslusser** [comment]
> Can you add the doc to `mkdocs.yml` so it appears in the docs site? Sorry I forgot to mention that in the original review.

done

**Review by joeriddles — APPROVED** [review]

**Pull request merged**